specs/templates/component-interaction-design.md

```diff
@@ -0,0 +1,23 @@
+# Interaction Design (IxD) [Template]
+
+## Problem Statement
+
+*Brief description of the reason for this work and an executive summary of the proposed design.*
+
+## Links To Relevant Work Items and Reference Material
+
+*Include links to the work items this design addresses.*
+*Include links to any other meaningful reference material.*
```

Comment:
```suggestion
*Relevant historical or background information; link to UX issue implementation specification, etc.*
```

Reply:
I currently have the following, which works for me… let me know if you disagree.

```
*Relevant historical or background information*
*Link to Interaction Design issue*
*Link to relevant work items, related existing issues, etc.*
```